CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # AppSignal for Node.js Changelog
 
+## 3.0.0
+
+Welcome to AppSignal for Node.js version 3.0.0.!
+
+In this release, we have switched to using OpenTelemetry internally for tracing. (We do not yet support OpenTelemetry for metrics and logging, but do continue to support metrics through our own metrics system.) It is no longer needed to use AppSignal instrumentation packages like `@appsignal/express`, `@appsignal/koa`, `@appsignal/nextjs`, `@appsignal/apollo-server`, etc. Some official OpenTelemetry instrumentation packages are bundled with our package as dependencies and instrumented automatically. Other _supported instrumentation packages_ can be [added and configured manually](https://docs.appsignal.com/nodejs/3.x/instrumentation/additional-instrumentations.html).
```

Comment:
```suggestion
In this release, we have switched to using OpenTelemetry internally for tracing. (We do not yet support OpenTelemetry for metrics and logging, but do continue to support metrics through our own metrics system.) The AppSignal instrumentation packages `@appsignal/express`, `@appsignal/koa`, `@appsignal/nextjs` and `@appsignal/apollo-server` should no longer be used. Some official OpenTelemetry instrumentation packages are bundled with our package as dependencies and instrumented automatically. Other _supported instrumentation packages_ can be [added and configured manually](https://docs.appsignal.com/nodejs/3.x/instrumentation/additional-instrumentations.html).
```

Less of a "not needed" and more of a "please don't". Also, that's the whole list of packages.